client/dist/styles.css

```diff
@@ -1,7 +1,6 @@
 #app {
   background-color: lightgray;
   display: flex;
-  width: 60%;
```

Comment:
Yes, that width property needed to be revised. Maybe we can discuss soon the overall layout of the page in detail and decide on numbers